plugins/web/opentelemetry-instrumentation-document-load/src/utils.ts

```diff
@@ -60,12 +61,53 @@ export const getPerformanceNavigationEntries = (): PerformanceEntries => {
   return entries;
 };
 
+const vitalsMetricNames: Record<Metric['name'], string> = {
+  FCP: EventNames.FIRST_CONTENTFUL_PAINT,
+  FID: EventNames.FIRST_INPUT_DELAY,
+  TTFB: EventNames.TIME_TO_FIRST_BYTE,
+  LCP: EventNames.LARGEST_CONTENTFUL_PAINT,
+  CLS: EventNames.CUMULATIVE_LAYOUT_SHIFT
+}
+
 const performancePaintNames = {
   'first-paint': EventNames.FIRST_PAINT,
-  'first-contentful-paint': EventNames.FIRST_CONTENTFUL_PAINT,
 };
 
-export const addSpanPerformancePaintEvents = (span: Span) => {
+export const addSpanPerformancePaintEvents = (span: Span, callback: () => void) => {
+  const missedMetrics: Set<Metric['name']> = new Set(['FCP', 'FID', 'TTFB'])
+  if ('chrome' in globalThis) {
+    missedMetrics.add('LCP')
+    missedMetrics.add('CLS')
```

Comment:
Are those Chromium-specific? Is this documented?

Reply:
They are. It's documented in `web-vitals` NPM readme.